File: 프로그래머스/Level1/두 정수 사이의 합.js
// https://programmers.co.kr/learn/courses/30/lessons/12912

const solution = (a, b) => {
	const min = Math.min(a, b);
	const max = Math.max(a, b);
	let answer = 0;
	for (let i = min; i <= max; i++) {
		answer += i;
	}
	return answer;
};

File: 프로그래머스/Level1/두 정수 사이의 합.py
# https://programmers.co.kr/learn/courses/30/lessons/12912

def solution(a, b):
    def calSum(x, y):
        s = 0
        for i in range(x, y + 1):
            s += i
        return s
    if a == b:
        return a
    if a > b:
        return calSum(b, a)   
    else:
        return calSum(a, b)